crates/holochain/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Fixed
+
+- This release contains a coarse solution for the issues with installing happ bundles or registering DNA via the admin API concurrently. [#881](https://github.com/holochain/holochain/pull/881).
```

Comment:
@freesig would you say the solution is still coarse or is the wording too pessimistic now?

Reply:
No it solves this issue in a fine grained way but there is a larger issue of how we use tokio locks but that's separate from this defect

Reply:
okay, reworded